src/server/server.ts

```diff
@@ -131,9 +133,15 @@ app.get('/*', bodyParser.raw(), serveArticle);
 app.post('/article', bodyParser.raw(), serveArticlePost);
 
 const port = 3040;
-app.listen(port, () => {
-  logger.info(`Server listening on port ${port}!`);
-  if (process.env.NODE_ENV !== "production") {
-    logger.info(`Webpack dev server is listening on port 8080`);
-  }
-});
+
+fs.readFile('dist/assets/manifest.json', function(err, data) {
```

Comment:
Can we package the file within the server.js so we don't have to ship it? I seem to remember there's a way to convince webpack to import these

Reply:
Are you referring to [this](https://webpack.js.org/loaders/json-loader/)? Based on the note there it looks like it's now built-in.

Reply:
I think you can just do this in Node 
`const json = require('./manifest.json');`

Which I'll swap to. I initially didn't because it's synchronous, but the server will need it before starting anyway.

Reply:
node requires `--resolveJsonModule` so will try the webpack loader